### DISCUSSION TITLE
How to copy the additional vasp file to the compute node?

### DISCUSSION THREAD

**Discussion by ysu-007** [deepmodeling/dpgen#539] [Q&A] [answered]
Hi everyone, 
I use degen to train a potential. The DFT calculation is special, there is an additional file 'vdw_kernel.bindat' that has to be included in the DFT computation directory to calculate the Van der Waals forces.  How can I auto send the file from the dpdispatcher work directory to the compute node?
Thanks

**Comment by ysu-007** [comment, marked as answer]
I have seen a discussion about this.  https://github.com/deepmodeling/dpgen/discussions/436 
So is this function has been added in DPGEN now? @AnguseZhang

**Reply by AnguseZhang** [reply]
Yes, please search "user_forward_files" in manual.

**Reply by AnguseZhang** [reply]
https://github.com/deepmodeling/dpgen/blob/master/examples/CH4-refact-dpdispatcher/machine-ali-ehpc.json

**Reply by ysu-007** [reply]
Thanks

**Reply by ysu-007** [reply]
New problem! How to use this command in the init process of dpgen?  I set it in the PARAM, however, it doesn't work. Thanks

**Reply by AnguseZhang** [reply]
Same process, you should set it in `MACHINE`

**Reply by ysu-007** [reply]
I see what the problem is here. Thanks.
